lib/vagrant-service-manager/plugin_util.rb

```diff
@@ -111,6 +111,7 @@ def self.execute_and_exit_on_fail(machine, ui, command)
 
       def self.execute_once(machine, ui, command)
         machine.communicate.sudo(command) do |_, data|
+          next if data.chomp.empty?
```

Comment:
> Now, in case of Windows OS, the data part contains expected info on 2nd iteration 

This is the part I don't get, where is the loop here? There is no loop, right?